nano/node/nodeconfig.hpp

```diff
@@ -85,7 +85,7 @@ class node_config
 	std::chrono::seconds tcp_io_timeout{ (network_params.network.is_dev_network () && !is_sanitizer_build) ? std::chrono::seconds (5) : std::chrono::seconds (15) };
 	std::chrono::nanoseconds pow_sleep_interval{ 0 };
 	std::size_t active_elections_size{ 5000 };
-	std::size_t active_elections_hinted_limit{ 1000 };
+	std::size_t active_elections_hinted_limit_ratio{ 20 }; // Limit of hinted elections as percentage of active_elections_size
```

Comment:
Should active_elections_hinted_limit_ratio be called active_elections_hinted_limit_percent?